src/ucp/rndv/rndv.c

```diff
@@ -2024,13 +2027,11 @@ static void ucp_rndv_dump(ucp_worker_h worker, uct_am_trace_type_t type,
     const ucp_rndv_rtr_hdr_t *rndv_rtr_hdr = data;
     const ucp_rndv_data_hdr_t *rndv_data   = data;
     const ucp_reply_hdr_t *rep_hdr         = data;
-    UCS_STRING_BUFFER_ONSTACK(rts_info, 64);
+    UCS_STRING_BUFFER_ONSTACK(rts_info, 128);
     const void *rkey_buf;
 
     switch (id) {
     case UCP_AM_ID_RNDV_RTS:
-        ucs_assert(rndv_rts_hdr->sreq.ep_id != UCP_EP_ID_INVALID);
```

Comment:
why removed?

Reply:
since this is packet dump, no need to check it here